o3/hooks/hdfs_hook.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Copy of :py:mod:`airflow.hooks.hdfs_hook`, modified to replace Py3 incompatible
+snakebite library wrapping with libhdfs3 driver.
+"""
+
+from airflow.hooks.base_hook import BaseHook
+
+import hdfs3
```

Comment:
@carlba This was the thing that was so damn hard getting working without the Conda packages. If you find a neater solution, I'd be happy to know about it! I stopped using Anaconda +3 years ago in favor of standard PyPI and stuff. A shame having to go back...